pkgs/by-name/sc/scmutils/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  mitschemeX11,
+  writeShellScriptBin,
+}:
+stdenv.mkDerivation rec {
+  pname = "scmutils";
+  version = "20230902";
+
+  src = fetchurl {
+    url = "https://groups.csail.mit.edu/mac/users/gjs/6946/mechanics-system-installation/native-code/${pname}-src-${version}.tar.gz";
+    hash = "sha256-9/shOxoKwJ4uDTHmvXqhemgy3W+GUCmoqFm5e1t3W0M=";
+  };
+
+  buildInputs = [ mitschemeX11 ];
+
+  configurePhase = ''
+    ln -r -s kernel/ghelper-pro.scm kernel/ghelper.scm
+    ln -r -s solve/nnsolve.scm solve/solve.scm
+    substituteInPlace load.scm \
+      --replace-fail '/usr/local/scmutils/' "$out/lib/mit-scheme/"
+  '';
+
+  buildPhase = ''
+    echo '(load "compile")' | mit-scheme --no-init-file --batch-mode --interactive
+    echo '(load "load") (disk-save "edwin-mechanics.com")' | mit-scheme --no-init-file --batch-mode --interactive
+  '';
+
+  installPhase = ''
+    mkdir -p "$out/lib/mit-scheme/" "$out/share/scmutils" "$out/bin"
+    cp edwin-mechanics.com "$out/lib/mit-scheme/"
+    declare -r TARGET="$out/lib/mit-scheme/"
+    for SRC in $(find * -type f -name '*.bci'); do
+      install -d "$TARGET"scmutils/"$(dirname "$SRC")"
+      cp -a "$SRC" "$TARGET"scmutils/"$SRC"
+    done
+    # Convenience script to load the band
+    declare -r CMD="exec ${mitschemeX11}/bin/mit-scheme --band $out/lib/mit-scheme/edwin-mechanics.com"
+    echo "#!$SHELL" > $out/bin/scmutils
+    echo "$CMD" "\"\$@\"" >> $out/bin/scmutils
+    echo "#!$SHELL" > $out/bin/edwin-scmutils
+    echo "$CMD" "--edit" "\"\$@\"" >> $out/bin/edwin-scmutils
+    chmod uog+rx "$out/bin/scmutils" "$out/bin/edwin-scmutils"
+    ln -r -s "$out/bin/edwin-scmutils" "$out/bin/mechanics"
```

Comment:
```suggestion
    ln -r -s "$out/bin/edwin-scmutils" "$out/bin/mechanics"
    runHook postInstall
```

Reply:
I'll update the commit to include this change.